### PR TITLE
Updating feature tests with AC references

### PR DESCRIPTION
**Pull request by peterbarrow**
Updating the features tests for spots to reference the ACs from the spec.

### CODE REVIEW

File: core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Feature: Spot market
 
-  Scenario: 001 when an order is placed, holding accout should have the holding asset; when there is not enough asset to move to holding account, the order can not be placed. When trade happens, holding asset is released: 0080-SP-FEES-001; 0080-SP-FEES-002; 0080-SP-FEES-003; 0080-SP-FEES-004; 0080-SP-FEES-005; 0080-SP-FEES-006; 0080-SP-FEES-007
+  Scenario: 001 when an order is placed, holding account should have the holding asset; when there is not enough asset to move to holding account,
+            the order can not be placed. When trade happens, holding asset is released: 0080-SP-FEES-001; 0080-SP-FEES-002; 0080-SP-FEES-003;
+            0080-SP-FEES-004; 0080-SP-FEES-005; 0080-SP-FEES-006; 0080-SP-FEES-007
```
Comment (jeremyletang) [inline core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature:5]: this looks broken, isn't it?

File: core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature [hunk 2]
```diff
@@ -135,16 +138,19 @@ Feature: Spot market
...
-    #the maker fee + infra fee = 0.04*(1*20)=0.8, party4 will receive 20ETH and with fee 0.8ETH substracted, so party4 recived 19.2ETH
+    #the maker fee + infra fee = 0.04*(1*20)=0.8, party4 will receive 20ETH and with fee 0.8ETH subtracted, so party4 received 19.2ETH
     Then "party4" should have holding account balance of "100" for asset "BTC"
     Then "party4" should have general account balance of "1920" for asset "ETH"
     Then "party4" should have general account balance of "300" for asset "BTC"
 
-    #If the order is cancelled or the size is reduced through an order amendment, funds should be released from the `holding_account` and returned to the `general_account`:0080-SP-FEES-008
+    # If the order is cancelled or the size is reduced through an order amendment, 
+    # funds should be released from the `holding_account` and returned to the `general_account`
+    # 0080-SP-FEES-008 (0080-SPOT-007)
```
Comment (gordsport) [inline core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature:148]: should the AC reference be 0080-SPOT-008

File: core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature [hunk 3]
```diff
@@ -45,7 +47,7 @@ Feature: Spot market
       | party2 | BTC/ETH   | sell | 100    | 3000  | 0                | TYPE_LIMIT | TIF_GTC | party-order2    |
       | party1 | BTC/ETH   | buy  | 200    | 1000  | 0                | TYPE_LIMIT | TIF_GTC | party-order11   |
       | party2 | BTC/ETH   | sell | 100    | 9000  | 0                | TYPE_LIMIT | TIF_GTC | party-order12   |
-    #During opening auction, holding asset is transfered from general account to holding account: 0080-SP-AUC-001
+    #During opening auction, holding asset is transferred from general account to holding account: 0080-SP-AUC-001
```
Comment (gordsport) [inline core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature:50]: should the AC reference be 0080-SPOT-001

File: core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature (hunk 1 above)
Comment (peterbarrow) [reply core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature:5]: Those references were there before I touched it, not sure what they are linked to so I didn't touch them (other than formatting)

**Review by guoguojin — APPROVED** [review]

**Pull request merged**